jcl/source/windows/JclDebug.pas

```diff
@@ -273,6 +273,7 @@   TJclMapScanner = class(TJclAbstractMapParser)
     function ProcNameFromAddr(Addr: DWORD): string; overload;
     function ProcNameFromAddr(Addr: DWORD; out Offset: Integer): string; overload;
     function SourceNameFromAddr(Addr: DWORD): string;
+    function VAFromUnitAndProcName( UnitName, ProcName : string ): DWORD;
```

Comment:
Please respect the style guide: https://wiki.delphi-jedi.org/wiki/Project_JEDI_Delphi_Language_Style_Guide
Here, there should be no space after the opening parenthesis, no space before the closing parenthesis and there `const` specifier must be added to both parameters declarations